src/axom/quest/interface/signed_distance.cpp

```diff
@@ -51,25 +52,23 @@ static struct parameters_t
 {
   int dimension; /*!< the dimension, 2 or 3 */
 
-  int max_levels;         /*!< max levels of subdivision for the BVH */
-  int max_occupancy;      /*!< max occupancy per BVH bin */
   bool verbose;           /*!< logger verbosity */
   bool is_closed_surface; /*!< indicates if the input is a closed surface */
   bool use_shared_memory; /*!< use MPI-3 shared memory for the surface mesh */
   bool compute_sign;      /*!< indicates if sign should be computed */
+  int allocator_id; /*!< the allocator ID to create BVH with (-1 for default) */
```

Comment:
alignment of comment

Reply:
Seems like clang-format doesn't like that 😕

Reply:
weird.. it's fine.